tests/testthat/test-03_additionalLinters.R

```diff
@@ -32,27 +32,15 @@ test_that("double_space_linter detects double spaces correctly", {
 #   expect_equal(internal_function_linter()(inputCorrect) %>% length, 0)
 # })
 
-
-# test_that("setwd_linter", {
-#   # nolint start
-#   inputWrong <- list(filename = "An example object",
-#                      file_lines = c("setwd(path)",
-#                                     'setwd("path"',
-#                                     "  setwd(path)"))
-#   # nolint end
-#   inputCorrect <- list(filename = "An example object",
-#                        file_lines = c("# Comment with the word setwd",
-#                                       "#' Roxygen comment containing setwd",
-#                                       "some code 123 # comment with setwd("))
-#   expect_true(lapply(setwd_linter()(inputWrong),
-#                      function(x) class(x) == "lint") %>% unlist %>% all)
-#   expect_equal(setwd_linter()(inputWrong) %>% length,
-#                inputWrong$file_lines %>% length)
-#   expect_true(lapply(setwd_linter()(inputCorrect),
-#                      function(x) class(x) == "lint") %>% unlist %>% all)
-#   expect_equal(setwd_linter()(inputCorrect) %>% length, 0)
-# })
-
+test_that("setwd_linter detects problematic cases of `setwd()`", {
+  test_file <- system.file("testScript_setwd_linter.R", package = "INWTUtils")
+  expect_true(file.exists(test_file))
+  lint_results <- lint(test_file, linters = setwd_linter())
+  lint_row_numbers <- unlist(unique(lapply(lint_results, function(x) x$line_number)))
+  lint_col_numbers <- unlist(lapply(lint_results, function(x) x$column_number))
+  expect_contains(lint_row_numbers, 2:4)
```

Comment:
```suggestion
  expect_equal(lint_row_numbers, 2:4)
```
Mir fiel gerade auf, dass der Test sonst nicht merkt, falls - zu Unrecht - noch weitere Zeilen als falsch identifiziert werden.